Translation corrections (#142)
07th-mod/higurashi-console-arcs: Update/some_013.txt

//rおかーさんがおばーちゃんを口論の末に殺して、私とおとーさんがそれを隠すことに協力する。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "おかーさんがおばーちゃんを口論の末に殺して、私とおとーさんがそれを隠すことに協力する。",
			NULL, "Mom killed Grandma after a lengthy argument, then Dad and I cooperated to hide the body.", Line_Normal);
	ClearMessage();

//rでも、それが見つかって猜疑にかられたおかーさんはおとーさんを殺し、続いて私を殺そうと、包丁を振りかざして襲ってくる…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも、それが見つかって猜疑にかられたおかーさんはおとーさんを殺し、続いて私を殺そうと、包丁を振りかざして襲ってくる…。",

> 			NULL, "But, when it was discovered, Mom was killed Dad and tried to kill me in a paranoid fit, attacking with a kitchen knife...", Line_Normal);

Reviewer: ```suggestion
			NULL, "But, when it was discovered, Mom killed Dad and tried to kill me in a fit of paranoia, attacking with a kitchen knife...", Line_Normal);
```

//rううん、…もう『公由』って苗字も、捨てよう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ううん、…もう『公由』って苗字も、捨てよう。",

> 			NULL, "No... I'll throw away the name of \"Kimiyoshi\" too.", Line_Normal);

Reviewer: ```suggestion
			NULL, "No... I'll throw away the name \"Kimiyoshi\" too.", Line_Normal);
```

//rさっき夢に見たショックが大きすぎて、まだ引きずっているだけなんだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "さっき夢に見たショックが大きすぎて、まだ引きずっているだけなんだ。",
			NULL, "My dream a little while ago was really shocking, so it must still be haunting me, that's all.", Line_Normal);
	ClearMessage();

//rそう、…ありえない。ありえないでしょ？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう、…ありえない。ありえないでしょ？",
			NULL, "That's right... it's impossible. Isn't it?", Line_Normal);
	ClearMessage();

//r家族が殺されたのに、…私だけ、生き残っているなんて。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "家族が殺されたのに、…私だけ、生き残っているなんて。",
			NULL, "For my whole family to have been killed... and me being the sole survivor.", Line_Normal);
	ClearMessage();

//r普通、一家惨殺なんて事件が起こったら、私もそれに巻き込まれるはずだよね？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "普通、一家惨殺なんて事件が起こったら、私もそれに巻き込まれるはずだよね？",

> 			NULL, "You'd think that if my whole family was murdered, I'd be no exception to that, would I?", Line_Normal);

Reviewer: ```suggestion
			NULL, "You'd think that if my whole family was murdered, I'd be no exception to that, wouldn't I?", Line_Normal);
```
